### ISSUE TITLE
Live loader and bulk loader incorrectly loads JSON without "uid" fields

### ISSUE THREAD

**Issue by danielmai** [dgraph-io/dgraph#3793]
### What version of Dgraph are you using?

master (bulk and live loader) and v1.0.16 (bulk loader only, live loader does not support JSON loading)

```
Dgraph version   : v1.0.12-rc3-659-g885e2e228
Commit SHA-1     : 885e2e228
Commit timestamp : 2019-08-09 15:21:23 -0700
Branch           : master
Go version       : go1.12.7
```

### Have you tried reproducing the issue with latest release?
Yes

### Steps to reproduce the issue (command/config used to run Dgraph).

1. Run Dgraph
2. Save this JSON data to a file called data.json

```json
[
  {"user": "User_1"},
  {"user": "User_2"},
  {"user": "User_3"}
]
```

3. Run live loader. The command with Dgraph master is:

```
dgraph live -f data.json -a localhost:9180 -z localhost:5180
```

Query for the users:

```
{
  q(func: has(user)) {
    uid
    user
  }
}
```

The query response only gives back a single user (could be User_1, User_2, or User_3):

```
{
  "data": {
    "q": [
      {
        "uid": "0x1",
        "user": "User_1"
      }
    ]
  }
}
```

This also happens with bulk loader in both master and v1.0.16.

### Expected behaviour and actual result.

I expect all three users to be inserted.

If I add explicit UID blank nodes to the JSON data, then all the data is mutated as expected:

```json
[
    {"uid": "_:user1", "user": "User_1"},
    {"uid": "_:user2", "user": "User_2"},
    {"uid": "_:user3", "user": "User_3"}
]
```

**Comment by campoy** [comment]
This seems like a very serious issue if we're losing data.

Can you update on it, @danielmai? Wasn't this fixed already?

**Comment by danielmai** [comment]
Yes, this is fixed in v1.1.0 by https://github.com/dgraph-io/dgraph/pull/3795. After following the reproducible steps of loading the data, the query comes back with all the nodes from the JSON input now:

```
{
    "q": [
      {
        "uid": "0x1",
        "user": "User_1"
      },
      {
        "uid": "0x2711",
        "user": "User_2"
      },
      {
        "uid": "0x4e21",
        "user": "User_3"
      }
    ]
}
```
